packages/rich-text/src/indent-list-items.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Internal dependencies
+ */
+
+import { LINE_SEPARATOR } from './special-characters';
+import { normaliseFormats } from './normalise-formats';
+import { getLineIndex } from './get-line-index';
+
+/**
+ * Indents any selected list items if possible.
+ *
+ * @param {Object} value      Value to change.
```

Comment:
Do we have a type defined in JSDoc for rich text? If not, should we?

Reply:
No, not so far. Perhaps we can look into that separately.